src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp

```diff
@@ -1832,6 +1832,74 @@ GUI_TEST_CLASS_DEFINITION(test_4170) {
 }
 
+GUI_TEST_CLASS_DEFINITION(test_4177) {
+    class FontSettingsHelper {
+    public:
+        static void changeFontAndSize(HI::GUITestOpStatus &os,  const QString &fontFamilyStr, int fontSize) {
+            QComboBox *fontComboBox = GTWidget::findExactWidget<QComboBox *>(os, "fontComboBox");
+            GTComboBox::selectItemByText(os, fontComboBox, fontFamilyStr);
+            GTGlobals::sleep(500);
+            GTSpinBox::setValue(os, GTWidget::findExactWidget<QSpinBox *>(os, "fontSizeSpinBox"), fontSize, GTGlobals::UseMouse);
+        }
+
+        static void fontChecker(HI::GUITestOpStatus &os, const QString &expectedFamilyStr, int expectedSize) {
+            CHECK_SET_ERR(GTComboBox::getCurrentText(os, "fontComboBox") == expectedFamilyStr, "unexpected style: " + GTComboBox::getCurrentText(os, "fontComboBox"));
+            int actualSize = GTSpinBox::getValue(os, GTWidget::findExactWidget<QSpinBox *>(os, "fontSizeSpinBox"));
+            CHECK_SET_ERR(actualSize == expectedSize, QString("unexpected point size: %1").arg(QString::number(actualSize)));
+        }        
+        
+        static void getFontSettings(HI::GUITestOpStatus &os, QString &familyStr, int &size) {
+            familyStr = GTComboBox::getCurrentText(os, "fontComboBox");
+            size = GTSpinBox::getValue(os, GTWidget::findExactWidget<QSpinBox *>(os, "fontSizeSpinBox"));
+        }
+    };
+
+    GTFileDialog::openFile(os, dataDir + "samples/CLUSTALW/COI.aln");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    GTUtilsOptionPanelMsa::openTab(os, GTUtilsOptionPanelMsa::TreeSettings);
+    GTUtilsDialog::waitForDialog(os, new BuildTreeDialogFiller(os, "default", 0, 0, true));
+    GTWidget::click(os, GTWidget::findWidget(os, "BuildTreeButton"));
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    QWidget *labelsColorButton = GTWidget::findWidget(os, "labelsColorButton");
+    CHECK_SET_ERR(labelsColorButton != NULL, "labelsColorButton not found");
+    if (!labelsColorButton->isVisible()) {
+        GTWidget::click(os, GTWidget::findWidget(os, "lblFontSettings"));
+    }
+    QString defaultFontFamily;
+    int defaultSize;
+
+    QGraphicsView *treeView = qobject_cast<QGraphicsView *>(GTWidget::findWidget(os, "treeView"));
+    QList<GraphicsButtonItem *>  nodes = GTUtilsPhyTree::getOrderedRectangularNodes(os);
+    //1. Open samples/CLUSTALW/COI.aln and build tree for it
+    GTUtilsPhyTree::clickNode(os, nodes[0]);//drop sticked ruler
+    //2. Select node, change font size to 16, also remember default parameters
+    GTUtilsPhyTree::clickNode(os, nodes[1]);
+    FontSettingsHelper::getFontSettings(os, defaultFontFamily, defaultSize);
+    FontSettingsHelper::changeFontAndSize(os, defaultFontFamily, 16);
+    //3. Click on parent node for node selected at step 1.
+    //Change its font to Arial with size 22
+    GTUtilsPhyTree::clickNode(os, nodes[2]);
+    FontSettingsHelper::changeFontAndSize(os, "Arial", 22);
```

Comment:
Q: Linux/Mac may not have Arial by default. Does it stiil work?

Reply:
It should be on Mac by default. Almost sure about Ubuntu, we will see on after running GUI tests.